cmd20.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
+# Name: cmd20py
+# Authors: Luke Short | Joe R
+# Repo: https://github.com/ekultails/cmd20py/
+
+import random
+
+# Number Generator
+random.seed(None)
```

Comment:
This should become it's own method such as `random_generator()`. Then we can change the logic for how the random number is generated. For example, we may want to expand the functionality by using the [secrets](https://docs.python.org/3/library/secrets.html) library to always generate a new truly random seed.

Reply:
@ekultails check the PR below to see if it meets the request.

https://github.com/ekultails/cmd20py/pull/6

Thanks!